src/components/Navigation/Toolbar/Toolbar.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import classes from './Toolbar.module.css'
+
+const toolbar = () => (
+    <div className={classes.Toolbar}>
+      <p>Toolbar</p>
+    </div>
+  );
+
+export default toolbar;
```

Comment:
What about rename all component files to _index.js_ to avoid the _/Toolbar/Toolbar_ import everytime? (I don't feel good with this duplicated name :/)